src/ru/nudemoon/src/eu/kanade/tachiyomi/extension/ru/nudemoon/Nudemoon.kt

```diff
@@ -336,4 +348,23 @@ class Nudemoon : ParsedHttpSource() {
         Genre("titsfuck"),
         Genre("x-ray"),
     )
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
```

Comment:
consider implementing [this](https://github.com/keiyoushi/extensions-source/blob/68e73a297f229a7a7b2dc675f1aacc0e17e061f5/src/ru/rumix/src/eu/kanade/tachiyomi/extension/ru/rumix/RuMIX.kt#L51-L77) so extension update changing the domain actually changes it for people.